default.nix

```
@@ -35,14 +35,14 @@ let
       mkdir -p $out/bin
       echo "#!${pythonForThis}/bin/python" > $out/bin/jumpcutter
       cat $src/jumpcutter.py >> $out/bin/jumpcutter
-      substituteInPlace $out/bin/jumpcutter --replace ffmpeg ${ffmpeg}
+      substituteInPlace $out/bin/jumpcutter --replace "ffmpegPath = 'ffmpeg'" "ffmpegPath = '${ffmpeg}'"
```

See `jumpcutter.py`, updated to fit pep8 constant value naming convention.
```suggestion
      substituteInPlace $out/bin/jumpcutter --replace "FFMPEG_PATH = 'ffmpeg'" "FFMPEG_PATH = '${ffmpeg}'"
```